src/index.js

```diff
@@ -9,6 +12,26 @@ export function entitize(json) {
   return { entities };
 }
 
+export function findAssociated(store, resourceType, id, assocation) {
+  const chosenAssociation = get(
+    store,
+    ["entities", resourceType, id, "relationships", assocation],
+    {}
+  );
+
+  if (isArray(chosenAssociation)) {
+    return map(chosenAssociation, (association) => {
+      return findById(store, association.type, association.id);
+    });
+  } else {
+    return findById(store, chosenAssociation.type, chosenAssociation.id);
```

Comment:
This is a bit weird to me. So this can return either an `Array` or an `object`? Shouldn't it always return an array?